gerousiabot/bot_handlers.py

```diff
@@ -0,0 +1,80 @@
+from telegram import Update, InlineKeyboardButton, InlineKeyboardMarkup
+from telegram.ext import CommandHandler, MessageHandler, Filters, CallbackQueryHandler, CallbackContext
+
+
+def get_bot_handlers() -> list:
+    """
+    Gets the list of the handlers to be used by the bot
+    :return: the list of the handlers
+    """
+    start_handler = CommandHandler('start', start)
+    user_query_handler = CommandHandler('chkusers', check_users)
+    button_handler = CallbackQueryHandler(keyboard_button_pressed)
+    unknown_handler = MessageHandler(Filters.command, unknown_command)
+
+    # unknown command handler must be the last added handler or we run into logical errors
+    return [start_handler, button_handler, user_query_handler, unknown_handler]
+
+
+def start_keyboard() -> list:
```

Comment:
[Suggestion] It's helpful to place the method definitions underneath to where they're called. Helps with the readability